docs/docs/quickstart-inference.md

```diff
@@ -0,0 +1,283 @@
+
+Install the required libraries:
+
+```bash
+pip install weave openai
+```
+
+Get your W&B API key from [https://wandb.ai/authorize](https://wandb.ai/authorize).
+
+## Step 2: Make your first traced LLM call
+
+Start with a simple example that uses Llama 3.1 8B through W&B Inference:
+
+```python
+import weave
+import openai
+
+# Initialize Weave - replace with your-team/your-project
+weave.init("my-first-weave-project")
+
+# Create an OpenAI-compatible client pointing to W&B Inference
+client = openai.OpenAI(
+    base_url='https://api.inference.wandb.ai/v1',
+    api_key="YOUR_WANDB_API_KEY",  # Replace with your actual API key
+    project="my-first-weave-project",  # Required for usage tracking
+)
+
+# Decorate your function to enable tracing
+@weave.op()
+def ask_llama(question: str) -> str:
+    response = client.chat.completions.create(
+        model="meta-llama/Llama-3.1-8B-Instruct",
+        messages=[
+            {"role": "system", "content": "You are a helpful assistant."},
+            {"role": "user", "content": question}
+        ],
+    )
+    return response.choices[0].message.content
+
+# Call your function - Weave automatically traces everything
+result = ask_llama("What are the benefits of using W&B Weave for LLM development?")
+print(result)
+```
+
+When you run this code, Weave:
+- Traces your LLM call automatically
+- Logs inputs, outputs, latency, and token usage
+- Provides a link to view your trace in the Weave UI
+
+## Step 3: Build a text summarization application
+
+Create a simple summarization app that shows how Weave traces nested operations:
+
+```python
+@weave.op()
+def extract_key_points(text: str) -> list[str]:
+    """Extract key points from a text."""
+    response = client.chat.completions.create(
+        model="meta-llama/Llama-3.1-8B-Instruct",
+        messages=[
+            {"role": "system", "content": "Extract 3-5 key points from the text. Return each point on a new line."},
+            {"role": "user", "content": text}
+        ],
+    )
+    return response.choices[0].message.content.strip().split('\n')
+
+@weave.op()
+def create_summary(text: str, key_points: list[str]) -> str:
+    """Create a concise summary based on key points."""
+    points_text = "\n".join([f"- {point}" for point in key_points])
+    response = client.chat.completions.create(
+        model="meta-llama/Llama-3.1-8B-Instruct",
+        messages=[
+            {"role": "system", "content": "Create a 2-3 sentence summary based on these key points."},
+            {"role": "user", "content": f"Key points:\n{points_text}"}
+        ],
+    )
+    return response.choices[0].message.content
+
+@weave.op()
+def summarize_text(text: str) -> dict:
+    """Main summarization pipeline."""
+    key_points = extract_key_points(text)
+    summary = create_summary(text, key_points)
+    return {
+        "key_points": key_points,
+        "summary": summary
+    }
+
+# Try it with sample text
+sample_text = """
```

Comment:
This sample text in the middle of code could become less accurate over time. Maybe the sample text should not be about Weave itself.